src/app/rooms/rooms.component.ts

```diff
@@ -0,0 +1,28 @@
+import { Component, OnInit } from '@angular/core';
+import { Room } from '../room';
+import { RoomService } from '../room.service';
+
+/**
+ * This component gives you a overview of each room with it's id.
```

Comment:
dont use word like "use, I". This is describing the function of a component, not telling a story.
Just say something like, displays a list of rooms with its overview or something.

Reply:
Changed. See new update